config/settings.yml

```diff
@@ -1288,7 +1288,7 @@
       :connection_pool_size: 8
       :memory_threshold: 1.gigabytes
       :nice_delta: 1
-    :websocket_worker:
+    :remote_console_worker:
```

Comment:
I'd think we need a migration to walk their settings changes to see if they have any changes to the old websocket_worker role and change that change to the new name.    Similar as below for the role name.

Reply:
Done!